iml-agent/src/action_plugins/kernel_module.rs

```diff
@@ -0,0 +1,19 @@
+use crate::agent_error::{ImlAgentError, MyKmodError};
+
+use kmod;
+
+pub async fn loaded(module: String) -> Result<bool, ImlAgentError> {
+    let ctx = kmod::Context::new().map_err(|e| MyKmodError::from(e))?;
+
+    let mut result = false;
+    for m in ctx.modules_loaded().map_err(|e| MyKmodError::from(e))? {
+        let name = m.name();
+
+        if name == module {
```

Comment:
Maybe use something like `find(...).is_some()` with `ModuleIterator`?

Reply:
Sure, dropped the draft code